### PR TITLE
DPL: work with ROOT master

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

File: Framework/Core/src/RCombinedDS.cxx [hunk 1]
```diff
@@ -42,7 +42,9 @@ double loop on the entries of the two RDataSources.
 #endif
 
 #include <ROOT/TSeq.hxx>
+#if __has_include(<ROOT/RMakeUnique.hxx>)
 #include <ROOT/RMakeUnique.hxx>
```
Comment (pzhristov) [inline Framework/Core/src/RCombinedDS.cxx]: I propose simply to replace RMakeUnique.hxx by memory. With c++17 they are equivalent according to the comment and content of the file.

Comment (ktf) [reply Framework/Core/src/RCombinedDS.cxx]: I agree, but I could not find the suggestion. Where is it?

Comment (ktf) [reply Framework/Core/src/RCombinedDS.cxx]: Actually I suspect it was already done and I simply did not drop the include.

Comment (pzhristov) [reply Framework/Core/src/RCombinedDS.cxx]: See commit 8044f652ec11ffe96b9d8e1127f42dd2ce4a2524 in Root.

Comment (ktf) [reply Framework/Core/src/RCombinedDS.cxx]: Indeed it seems to work without the header....

**Comment by ktf** [comment]
Compiles fine where it matters. Merging.

**Pull request merged**